### ISSUE TITLE
[ISSUE] Handling of slashes in expiry date field.

### ISSUE THREAD

**Issue by DagAgren** [verygoodsecurity/vgs-collect-ios#150]
When using version 1.5.1 and entering a date in an expiry date field, after inputting two letters, they are shown as "12". When entering a third, a slash is added: "12/3".

It would be clearer to the user to add the / already after entering two letters: "12/".

**Comment by dmytrokhl** [comment]
@DagAgren this is good point, we will look if we can improve it.

**Comment by DagAgren** [comment]
This also applies to the spaces in the card number field, we just noticed. The cursor should always be placed at the location the next number will appear, so after typing "4444", the cursor should be placed as if the space had already been added, "4444 |"

**Comment by dmytrokhl** [comment]
Yes, this is same functionality under the hood. The change will effect any field that use default or custom `formatPattern`.
